student/src/main/java/tds/student/sql/repository/RemoteExamRepository.java

```diff
@@ -203,4 +213,35 @@ public void approveAccommodations(UUID examId, ApproveAccommodationsRequest appr
     
     return Optional.absent();
   }
+  
+  @Override
+  public Response<ExamConfiguration> startExam(UUID examId) throws ReturnStatusException {
+    HttpHeaders headers = new HttpHeaders();
+    headers.set("Accept", MediaType.APPLICATION_JSON_VALUE);
+    headers.setContentType(MediaType.APPLICATION_JSON);
+    HttpEntity<?> requestHttpEntity = new HttpEntity<>(headers);
+    Response<ExamConfiguration> response;
+    UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(String.format("%s/%s/start", examUrl, examId));
```

Comment:
Just throwing it out there, but String.format can do funky things in different locales... (especially with numbers)